internal/audit/kafka/conf.go

```diff
@@ -55,7 +55,7 @@ type Conf struct {
 	// Brokers list to seed the Kafka client.
 	Brokers []string `yaml:"brokers" conf:"required,example=['localhost:9092']"`
 	// Compression sets the compression algorithm to use in order of priority. Valid values are "none", "gzip", "snappy","lz4", "zstd". Default is ["snappy", "none"].
-	Compression []string `yaml:"compression" conf:",example=['snappy']"`
+	Compression []string `yaml:"compression" conf:",example=['zstd', 'snappy', 'none']"`
```

Comment:
If the default is `['snappy', 'none']`, let's set the example value to that as well.
```suggestion
	Compression []string `yaml:"compression" conf:",example=['snappy', 'none']"`
```